haskell-language-server.cabal

```diff
@@ -2027,7 +2027,7 @@ test-suite func-test
   type:               exitcode-stdio-1.0
   build-tool-depends:
     haskell-language-server:haskell-language-server,
-    ghcide:ghcide-test-preprocessor
+    -- ghcide:ghcide-test-preprocessor
```

Comment:
What's the reason for this change?

Reply:
Well spotted, I forget to update this part.

Reply:
fixed @fendor

Reply:
It is a bit weird, shouldn't the tests have failed? Is this maybe not even needed or are the tests dynamically skipped if they can't find `ghcide-test-preprocessor` on the `PATH`?

Reply:
I take a look into it. I think we can remove ghcide-test-preprocessor from func-test. Not even used. only user is ghcide-tests